package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-tumblr",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
Please don't change the version number, I have a release process that will take care of this.